static/javascripts/kp_extraction_emb_test.js

```diff
@@ -0,0 +1,84 @@
+function submit(input_text, number_keyphrases) {
+	$("#last_group").append('<i class="fa fa-spinner fa-spin" id="spinner"></i>');
+  $('#input_text').val(input_text);
+  
+	url = "http://52.30.254.33:1277/api/kp";
+	data = {'inp_url' : input_text,
+	  'nbkp' : number_keyphrases
+	  };
+$.ajax({
+  url: "http://52.30.254.33:1277/api/kp",
+  type: "POST",
+  contentType: 'application/json',
+  dataType :"json",
```

Comment:
Check everywhere that you have spaces between symbols. Use the auto format method from your IDE otherwise, the whoel code is wrongly formatted right now

Reply:
Okay I'll auto format it